scripts/Invoke-SasWorkstationProvisioner.ps1

```
@@ -0,0 +1,218 @@
+    if ($parent) { New-Item -ItemType Directory -Path $parent -Force | Out-Null }
+    [System.IO.File]::WriteAllText($Path, (($Value | ConvertTo-Json -Depth 12) + "`n"), [System.Text.UTF8Encoding]::new($false))
+}
+
+function Test-PowerShell7 {
+    $pwsh = Get-Command pwsh -ErrorAction SilentlyContinue
+    return ($null -ne $pwsh)
+}
+
+function Test-WslCapability {
+    try {
+        $result = & wsl.exe --status 2>&1
```

PSScriptAnalyzer (1.25.0)

[warning] The variable 'result' is assigned but never used.

(PSUseDeclaredVarsMoreThanAssignments)